### PR TITLE
interface: skin cleanup

### PR DESCRIPTION
**Pull request by quardbreak**
## Description of changes
изображение

* Cleanups skin.dmf and interface related things:
  * All skins elements renamed in flavor of name standardization.
  * Fixes weird elements offset on the right side of the screen.
  * Most buttons are gone now, only four left.
изображение
  * Replaces default status bar with alternative implementation using label element.
  * Prefix in the info panel use blue color now.

* Not skin related changes:
  * Kills discord button.
  * Renames forum button to community.
  * Shuffles Show Server Revision. Now it prints everything, but if server don't have the repository link set, the link won't be created. The output style will look like `branch-example | 2022-04-28 | hash`.
изображение
  * Fixes tooltips offsets. Removes clockcult theme. Updates jquery script link.

## Why and what will this PR improve
I wanted to redesign interface to be more Antimonium-like fullscreen interface with dark layout without borders, yet I'm only able to cleanup things right now!

## Authorship
tgstation: status bar label code.
me: everything else.

### CODE REVIEW

**Comment by MistakeNot4892** [comment]
Personally I really don't like the checkboxes, or how flat the UI looks. The buttons at the top are crammed together and it all generally looks a bit gross.

File: code/controllers/configuration.dm [hunk 1]
```diff
@@ -244,7 +243,7 @@ var/global/list/gamemode_cache = list()
 	var/act_interval = 0.1 SECONDS //Interval for spam prevention.
 
 	var/panic_bunker = FALSE //is the panic bunker enabled?
-	var/panic_bunker_message = "Sorry! The panic bunker is enabled. Please head to our Discord or forum to get yourself added to the panic bunker bypass."
+	var/panic_bunker_message = "Sorry! The panic bunker is enabled. Please head to our staff to get yourself added to the panic bunker bypass."
```
Comment (MistakeNot4892) [inline code/controllers/configuration.dm]: `contact an admin` might be better than `head to our staff`.

File: code/game/world.dm [hunk 2]
```diff
@@ -214,8 +214,8 @@ var/global/world_topic_last = world.timeofday
 /world/proc/update_status()
 	var/s = "<b>[station_name()]</b>"
 
-	if(config && config.discordurl)
-		s += " (<a href=\"[config.discordurl]\">Discord</a>)"
+	if(config && config.forumurl)
```
Comment (MistakeNot4892) [inline code/game/world.dm]: `communityurl` may be better than `forumurl` if it's meant to do double duty as Discord link as welel.

**Comment by afterthought2** [comment]
The verdict here is likely going to be:
- Separate your functional changes into small discrete PRs so they can be evaluated individually. Some, especially the fixes, will be fine; some I am less optimistic about.
- The massive element rename is not going through; it's unnecessary and due to dm's compile-time checking abilities is really hard to control for quality.

**Comment by MistakeNot4892** [comment]
Given the maintainer view above and that quard has dropped off the Discord I'm going to close this for now, it can be reopened when split up.

**Pull request closed without merging**